### ISSUE TITLE
Missing dependency of cordova-plugin-splashscreen in plugin.xml

### ISSUE THREAD

**Issue by timbru31** [ionic-team/cordova-plugin-ionic#15]
It seems that the `cordova-plugin-splashscreen` is not listed as a dependency in the plugin.xml, therefore the installation fails, if the plugin is not present.

Ionic info
```
❯ ionic info
[ERROR] Error with ./node_modules/ionic-angular/package.json file: FILE_NOT_FOUND
[ERROR] Error with ./node_modules/@ionic/app-scripts/package.json file: FILE_NOT_FOUND

cli packages: (/Users/timbru/work/ume/door-to-door-webapp/node_modules)

    @ionic/cli-utils  : 1.12.0
    ionic (Ionic CLI) : 3.12.0

local packages:

    @ionic/app-scripts : not installed
    Ionic Framework    : not installed

System:

    Node : v6.11.4
    npm  : 3.10.10
    OS   : macOS High Sierra

Misc:

    backend : pro
```

Cordova information:

```
❯ cordova -v
7.1.0
```

**Comment by timbru31** [comment]
Follow up: after installing the plugin (`cordova plugin add cordova-plugin-splashscreen`) the build worked again (`cordova build ios --emulator --target "iPad-Air-2"`)

**Comment by Fuiste** [comment]
Whoops, seems to have been an oversight on my part there.  I've fixed the dependency tag and released version 2.0.1 of the plugin to address this.

Sorry for the inconvenience there!
